### PR TITLE
refactor(deps): remove bower

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,30 +2,12 @@
...
   "dependencies": {
-    "Duration.js": "duration.js#3.6.0",
-    "backbone": "1.1.1",
-    "blanket": "1.1.5",
-    "blueimp-canvas-to-blob": "2.1.0",
-    "chai": "1.8.1",
-    "cocktail": "0.5.9",
-    "easteregg": "https://github.com/mozilla/fxa-easter-egg.git#ab20cd517cf8ae9feee115e48745189d28e13bc3",
-    "es6-promise": "https://github.com/stefanpenner/es6-promise.git#4.1.1",
     "fxa-checkbox": "mozilla/fxa-checkbox#7f856afffd394a144f718e28e6fb79092d6ccddd",
```
Comment (vladikoff) [inline bower.json]: CSS, needs a bit more changes

File: bower.json [hunk 2]
```diff
@@ -2,30 +2,12 @@
...
-    "fxa-crypto-relier": "https://github.com/mozilla/fxa-crypto-relier.git#v2.1.0",
     "html5shiv": "3.7.2",
```
Comment (vladikoff) [inline bower.json]: Included in templates, need a bit more changes

File: bower.json [hunk 3]
```diff
@@ -2,30 +2,12 @@
...
-    "jquery": "3.1.0",
-    "jquery-modal": "https://github.com/shane-tomlinson/jquery-modal.git#0576775d1b4590314b114386019f4c7421c77503",
-    "jquery-simulate": "1.0.1",
-    "jquery-ui": "1.11.0",
     "jquery-ui-touch-punch": "https://github.com/zaach/jquery-ui-touch-punch.git#89e5a9159494de37df1e09cccf9036dda5429830",
```
Comment (vladikoff) [inline bower.json]: no pkg json, needs a tweak

File: bower.json [hunk 4]
```diff
@@ -2,30 +2,12 @@
...
-    "js-md5": "md5#0.6.0",
-    "mailcheck": "https://github.com/mailcheck/mailcheck.git#d995fcc1d02c124a893c2cb0fce2e0ce5f36bce4",
-    "mocha-blanketjs-adapter": "1.0.0",
-    "node-uuid": "1.4.3",
     "normalize-css": "2.1.3",
```
Comment (vladikoff) [inline bower.json]: CSS, needs a bit more changes

File: bower.json [hunk 5]
```diff
@@ -2,30 +2,12 @@
...
     "raven-js": "https://github.com/vladikoff/raven-js.git#customEndpoint-3.13.0",
```
Comment (vladikoff) [inline bower.json]: Already raven in the server, need to figure this out

File: bower.json [hunk 6]
```diff
@@ -2,30 +2,12 @@
...
     "sinon": "http://sinonjs.org/releases/sinon-3.2.1.js",
```
Comment (vladikoff) [inline bower.json]: Already sinon in the server, need to figure this out

File: bower.json [hunk 7]
```diff
@@ -2,30 +2,12 @@
...
-    "tos-pp": "https://github.com/mozilla/legal-docs.git",
     "ua-parser-js": "https://github.com/vladikoff/ua-parser-js.git#643d1698aef5bed095e1264ae258902bf346175c",
     "vat": "0.0.9",
```
Comment (vladikoff) [inline bower.json]: not on npm or something

File: bower.json [hunk 8]
```diff
@@ -2,30 +2,12 @@
...
     "webrtc-adapter": "0.2.5"
```
Comment (vladikoff) [inline bower.json]: older version not on npm

File: bower.json [hunk 9]
```diff
@@ -2,30 +2,12 @@
...
     "jquery-ui-touch-punch": "https://github.com/zaach/jquery-ui-touch-punch.git#89e5a9159494de37df1e09cccf9036dda5429830",
-    "js-md5": "md5#0.6.0",
-    "mailcheck": "https://github.com/mailcheck/mailcheck.git#d995fcc1d02c124a893c2cb0fce2e0ce5f36bce4",
-    "mocha-blanketjs-adapter": "1.0.0",
-    "node-uuid": "1.4.3",
     "normalize-css": "2.1.3",
     "raven-js": "https://github.com/vladikoff/raven-js.git#customEndpoint-3.13.0",
     "sinon": "http://sinonjs.org/releases/sinon-3.2.1.js",
-    "tos-pp": "https://github.com/mozilla/legal-docs.git",
     "ua-parser-js": "https://github.com/vladikoff/ua-parser-js.git#643d1698aef5bed095e1264ae258902bf346175c",
```
Comment (vladikoff) [inline bower.json]: todo

**Comment by vladikoff** [comment]
@vbudhram @philbooth r?

File: package.json [hunk 10]
```diff
@@ -76,32 +83,47 @@
...
     "serve-static": "1.13.1",
     "speed-trap": "git://github.com/rum-diary/speed-trap#0.0.7",
     "time-grunt": "1.4.0",
+    "ua-parser-js": "git://github.com/vladikoff/ua-parser-js.git#fxa-version",
     "uglifyjs-webpack-plugin": "1.1.8",
     "underscore": "1.8.3",
     "universal-analytics": "0.4.15",
     "webpack": "3.10.0",
-    "webpack-dev-middleware": "2.0.4"
+    "webpack-dev-middleware": "2.0.4",
```
Comment (vbudhram) [inline package.json]: Should this be a dev dependency? Docs [recommend](https://github.com/webpack/webpack-dev-middleware) installing it as that.

Comment (vladikoff) [reply package.json]: @vbudhram I shall check that!

**Pull request merged**